testutils/ttn.py

```diff
@@ -50,7 +47,7 @@ def __init__(self):
     def __enter__(self):
         self.mqtt.user_data_set(self)
         self.mqtt.tls_set()
-        password = get_required_envvar("LORAWAN_DL_KEY")
+        password = get_required_envvar("TTN_DL_KEY")
```

Comment:
... by doing something like this:

```suggestion
        try:
            password = get_required_envvar("TTN_DL_KEY")
        except RuntimeError:
            password = get_required_envvar("LORAWAN_DL_KEY")
```

Once the release tests are migrated (and backported to all desired releases) we can go for your final solution.